tests/vm/tcompiletimetable.nim

```diff
@@ -47,3 +50,9 @@ addStuff("Hey"): echo "Hey"
 addStuff("Hi"): echo "Hi"
 dump()
 
+# ensure .compileTime vars can be used at runtime:
+import macros
```

Comment:
```suggestion
```

The `import macros` is repeated, [macros is imported twice on the same file.](https://github.com/nim-lang/Nim/blob/f5d741d452c42140b729f4c598c10b177e9fda5e/tests/vm/tcompiletimetable.nim#L14) :slightly_smiling_face: